Observing-Interface-Setup/dds-interface.rst

```diff
@@ -0,0 +1,44 @@
+#############
+Debugging DDS
+#############
+
+Overview
+^^^^^^^^
+This topic describes procedures for debugging the status of the `DDS interface <https://ts-sal.lsst.io>`_ that you run on your own nublado that powers the :doc:`CSCs <obs-controls:System-Architecture/CSC-Overview/index>` that run on the summit.
```

Comment:
Not sure I understand what this phrase means. Is it nublado or CSC? We don't run CSCs on nublado so these are not exactly compatible. I think we are talking about  different things here. Which one are you trying to document?